test/dlc_tests/benchmark/ec2/pytorch/inference/test_performance_torch_inductor_inference.py

```diff
@@ -0,0 +1,177 @@
+# language governing permissions and limitations under the License.
+
+import os
+import time
+import pytest
+import re
+import pandas as pd
+import boto3
+import datetime
+from botocore.exceptions import ClientError
+from test.test_utils import (
+    CONTAINER_TESTS_PREFIX,
+    get_framework_and_version_from_tag,
+    LOGGER,
+    BENCHMARK_RESULTS_S3_BUCKET
+)
+
+SETUP_CMD = "cd $HOME && \
+             git clone --branch v2.0.0-rc3 --recursive --single-branch -depth 1 https://github.com/pytorch/pytorch.git && \
+             git clone --recursive https://github.com/pytorch/benchmark.git && \
+             git checkout $(cat pytorch/.github/ci_commit_pins/benchmark.txt && \
+             cd $HOME/benchmark && \
+             python install.py;"
+
+def unique_metric_dims(instance_type, precision, model_suite):
+    dimensions = [
+        {"Name": "InstanceType", "Value": instance_type},
+        {"Name": "ModelSuite", "Value": model_suite},
+        {"Name": "Precision", "Value": precision},
+        {"Name": "WorkLoad", "Value": "Inference"},
+    ]
+    return dimensions
+
+def get_boto3_session(region="us-east-1"):
+    """Get boto3 session with us-east-1 as default region used to connect to AWS services."""
+    return boto3.session.Session(region_name=region)
+
+def get_cloudwatch_client(region="us-east-1"):
+    """Get AWS CloudWatch client object. Currently assume region is IAD (us-east-1)"""
+    return get_boto3_session(region=region).client("cloudwatch")
+
+def put_metric_data(metric_name, namespace, unit, value, dimensions):
+    """Puts data points to cloudwatch metrics"""
+    cloudwatch_client = get_cloudwatch_client()
+    current_timestamp = datetime.datetime.utcnow()
+    try:
+        response = cloudwatch_client.put_metric_data(
+            Namespace=namespace,
+            MetricData=[
+                {
+                    "MetricName": metric_name,
+                    "Dimensions": dimensions,
+                    "Value": value,
+                    "Unit": unit,
+                    "Timestamp": current_timestamp,
+                }
+            ],
+        )
+    except ClientError as e:
+        LOGGER.error(
+            "Error: Cannot put data to cloudwatch metric: {}".format(metric_name))
+        LOGGER.error("Exception: {}".format(e))
+        raise e
+
+def read_metric(model_suite, csv_file):
+    csv = os.path.join("./", f"logs_{model_suite}", csv_file)
+    df = pd.read_csv(csv)
+    value = df[df.columns[-1]].iloc[0]
+    if isinstance(value, str):
+        for i in range(len(value)):
+            if value[i].isdecimal() or value[i] == ".":
+                continue
+            else:
+                return float(value[:i])
+    return float(value)
+
+def upload_metric(instance_type, precision, suite, metric_name, value, unit):
+    put_metric_data(
+        metric_name=metric_name,
+        namespace=f"pytorch/{instance_type}/Benchmarks/TorchDynamo/Inductor",
```

Comment:
instance_type is already part of dimensions. Remove from here since we want metrics from all benchmarks to end up in the same namespace for comparison

Reply:
You might instead want to differentiate ec2 vs SM here.

Reply:
Let's use PyTorch/EC2/Benchmarks/TorchDynamo/Inductor. I am syncing this across Training and Inference PRs.